easybuild/easyconfigs/p/PETSc/PETSc_remove-mpiicc-show-quotes.patch

```diff
@@ -0,0 +1,12 @@
+diff -Nru config/PETSc/Configure.py.back config/PETSc/Configure.py`
```

Comment:
Could you please add a line of description of what this patch fixes (and who the author is (you presumably))